.cirrus.yml

```diff
@@ -142,10 +142,29 @@ build_task:
     - source cirrus-env BUILD
     - source .cirrus/use-gradle-wrapper.sh
     - go version
-    - regular_gradle_build_deploy_analyze --profile
+    - regular_gradle_build_deploy_analyze --profile -x test -x sonar
```

Comment:
I like the idea of splitting the build and the test, this should speedup the whole process by parallelizing test and ITs.
Also, it reduces a lot of the logs in the build-only  phase (from 9'000+ lines to 3'000).